lib/api/2.0/nodes.js

```diff
@@ -61,24 +62,45 @@ var nodesGetPollersById = controller(function(req) {
 });
 
 var nodesGetWorkflowById = controller(function(req) {
-    return nodes.getNodeWorkflowById(req.swagger.params.identifier.value);
+    return workflowApiService.getWorkflowsByNodeId(req.swagger.params.identifier.value,
```

Comment:
@benbp So, I think your concern is valid, and I'm fine with changing this query for now.  The problem we have is that Cassandra does not support AND queries, so it's not something that can easily be abstracted away by the ORM.  Effectively we'd be doing the same thing just in a different place.  This will probably require changes to the models themselves to allow a direct query of active workflows associated with a node.